app/lib/presentation/features/match.dart

```diff
@@ -276,21 +288,29 @@ class _TeamsScreenState extends ConsumerState<_TeamsScreen> {
       builder: (context, state, _) {
         final values = state.value;
         return Column(
-          children: values.entries.map((e) {
-            return CupertinoFormRow(
-              prefix: Text(e.key.username),
-              child: CupertinoSlidingSegmentedControl<Team>(
-                groupValue: e.value,
-                onValueChanged: (team) =>
-                    _players.updateValue({...values, e.key: team ?? Team.none}),
-                children: const {
-                  Team.left: Text('Left'),
-                  Team.none: Text('None'),
-                  Team.right: Text('Right'),
-                },
-              ),
-            );
-          }).toList(),
+          children: values.entries.map(
+            (e) {
+              return CupertinoFormRow(
+                prefix: Text(e.key.username),
+                child: CupertinoSlidingSegmentedControl<Team>(
+                  groupValue: e.value,
+                  onValueChanged: (team) =>
+                      _players.updateValue({...values, e.key: team ?? Team.none}),
+                  children: {
+                    Team.left: Text(
+                      'RED',
+                      style: TextStyle(color: Colors.red),
+                    ),
+                    Team.none: const Text('None'),
+                    Team.right: Text(
+                      'BLUE',
+                      style: TextStyle(color: Colors.blue),
+                    ),
+                  },
+                ),
+              );
+            },
```

Comment:
```suggestion
            }
```
Rebase e già che ci sei toglimi questa virgola XD